packages/dapp/src/hooks/useEncryptionAccount.ts

```diff
@@ -0,0 +1,22 @@
+import { useState, useCallback } from 'react';
+
+export type UseEncryptionAccountHook = [
+  encryptionAccount: string,
+  setEncryptionAccount: Function
+];
+
+let initialEncryptionAccount = localStorage.getItem("encryptionAccount") as string;
+if (!initialEncryptionAccount) {
+  initialEncryptionAccount = "";
+}
+
+export const useEncryptionAccount = (): UseEncryptionAccountHook => {
+  const [encryptionAccount, setEncryptionAccount] = useState(initialEncryptionAccount);
+
+  const switchEncryptionAccount = useCallback((newEncryptionAccount) => {
```

Comment:
the same as in the https://github.com/windingtree/org.id-profile-manager/pull/3/files#r796513902